### ISSUE TITLE
Open Issues with Web Service

### ISSUE THREAD

**Issue by ReinhardKeil** [ARM-software/vscode-cmsis-csolution#147]
The CMSIS Solution obtains information about Boards and Devices via Web Services.  This workflow downloads packs seamless, but has currently a few flaws:

- Device specific templates are not listed, when the DFP is not installed on the local computer. Instead the build-in default templates are shown.
- Reference applications are not shown when the BSP is not installed on the local computer as there is not information about layers in the BSP obtained by the Web Service. (#117)
- Wrong vendor names used for boards provided via Solar search (#132)

Currently there is no way to list layers that are available in packs that are not downloaded.  Only after installation of a pack, the `csolution list layers` command is available.

**Comment by jkrech** [comment]
This issue https://github.com/ARM-software/vscode-cmsis-csolution/issues/117 is related

**Comment by ReinhardKeil** [comment]
I believe the workarounds are OK.  IMHO it is sufficient to just list examples for BSPs that are not installed.

**Comment by ReinhardKeil** [comment]
ticket is parked for future enhancements - currently no action planned

**Comment by jkrech** [comment]
At this point no enhancements are planned to the web service and we work with the available data.
